src/main/java/com/based/basedsurvey/controller/FillSurveyController.java

```diff
@@ -121,8 +122,9 @@ private void addResponse(Question q, String value) {
      * @return the HTML inputs for the question
      */
     private String getMultipleChoiceInputs(MultiplechoiceQuestion q) {
-        String s = "";
+        String s = "<hr>";
         s += "<h3> " + q.getPrompt() + " </h3>";
```

Comment:
https://github.com/wiverson/htmx-demo/blob/main/src/main/java/com/devhow/htmxdemo/InfiniteScroll.java#L47
You can make your inline html easier to read using an annotation. Heres an example.
Another way would be to use a thymeleaf block i think

Reply:
Just added the inline html annotations